pysb/bng.py

```diff
@@ -49,14 +52,25 @@ class BngBaseInterface(object):
 
     @abc.abstractmethod
     def __init__(self, model=None, verbose=False, cleanup=False,
-                 output_prefix=None, output_dir=None):
+                 output_prefix=None, output_dir=None,
+                 model_additional_species=None,
+                 model_population_maps=None):
+        self._logger = get_logger(__name__,
+                                  model=model,
+                                  log_level=verbose)
+        if model:
```

Comment:
The Interface can be called with or without a `model` object. Most use cases involve a PySB model which is loaded into BNG, but sometimes it is useful to load a BNGL file directly and use the interface as a "remote control" for BNG without a PySB model. The original motivation was the BNGL importer, which does the latter: https://github.com/pysb/pysb/blob/v1.4.0/pysb/importers/bngl.py#L30-L38